docs/release_notes/0.0.75.md

```diff
@@ -10,5 +10,8 @@ KM449 🐛 Make it possible to import okctl as a library
 
 KM406 👌 Use latest okctl github release as okctl development version (#765)
 
+KM445 👌 Add validations for okctl vs cluster version (#786).
```

Comment:
Good brief introduction. I suggest we flag it as a breaking change as well when releasing.

Reply:
Yeah, I'm thinking writing a heads up in Slack as well.